api/services/PassportService.js

```diff
@@ -203,11 +203,12 @@ module.exports = class PassportService extends Service {
 
     criteria[fieldName] = identifier
 
-    return this.app.services.FootprintService.find('User', criteria, {populate: 'passports', findOne: true})
+    return this.app.services.FootprintService.find('User', criteria, {populate: 'passports'})
```

Comment:
I'm really curious to see this and see how it will work... As I don't know anything about GraphQL for now, it a blur :)